<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTabs</class>
 <widget class="QTabWidget" name="MainTabs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::South</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="currentIndex">
   <number>7</number>
  </property>
  <widget class="QWidget" name="tab_activate">
   <attribute name="title">
    <string>Activate</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_activate">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Select coins for activation</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="2">
        <widget class="QCheckBox" name="checkBox_rvn">
         <property name="text">
          <string>Raven</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QCheckBox" name="checkBox_bat">
         <property name="text">
          <string>Basic Attention Token</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBox_ltc">
         <property name="text">
          <string>Litecoin</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="checkBox_zec">
         <property name="text">
          <string>Zcash</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="checkBox_rick">
         <property name="text">
          <string>Rick</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="checkBox_dai">
         <property name="text">
          <string>DAI</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="checkBox_morty">
         <property name="text">
          <string>Morty</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBox_dash">
         <property name="text">
          <string>Dash</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBox_kmd">
         <property name="text">
          <string>Komodo</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBox_eth">
         <property name="text">
          <string>Ethereum</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBox_btc">
         <property name="text">
          <string>Bitcoin</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="checkBox_labs">
         <property name="text">
          <string>Labs</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="checkBox_doge">
         <property name="text">
          <string>Dogecoin</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBox_bch">
         <property name="text">
          <string>Bitcoin Cash</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_balances">
   <attribute name="title">
    <string>Balances</string>
   </attribute>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>871</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Todo: list activated coins, with address, balance and fiat/btc value</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_myorders">
   <attribute name="title">
    <string>My Orders</string>
   </attribute>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>360</y>
      <width>531</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Shows current orders on the book</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_orderbook">
   <attribute name="title">
    <string>Orderbook</string>
   </attribute>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Shows orderbooks for activated coins</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>521</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>table data. filter, sort, export to csv</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>341</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>caching recommended</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_wallet">
   <attribute name="title">
    <string>Wallet</string>
   </attribute>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>751</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Send/recieve/balance for specific coin. QR code feat req.</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_trades">
   <attribute name="title">
    <string>Trades</string>
   </attribute>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>661</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Trade history table. Filter, sort, export to csv</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>440</y>
      <width>841</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Show status of pending swaps, and reason for failed swaps.</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_config">
   <attribute name="title">
    <string>Config</string>
   </attribute>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>361</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>MM2.json value inputs</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_log">
   <attribute name="title">
    <string>Logs</string>
   </attribute>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>471</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>stdout from mm2 console logs</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
